plugin/trino-example-plugin/pom.xml

```diff
@@ -0,0 +1,274 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>trino-example-plugin</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <packaging>trino-plugin</packaging>
+
+    <parent>
+        <groupId>io.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>128</version>
+    </parent>
+
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <project.build.targetJdk>11</project.build.targetJdk>
+
+        <air.main.basedir>\${project.basedir}</air.main.basedir>
+
+        <air.check.skip-extended>true</air.check.skip-extended>
+        <air.check.skip-license>false</air.check.skip-license>
+
+        <air.check.fail-checkstyle>true</air.check.fail-checkstyle>
+        <air.check.skip-checkstyle>false</air.check.skip-checkstyle>
+
+        <dep.trino.version>391</dep.trino.version>
+        <dep.airlift.version>217</dep.airlift.version>
+        <dep.packaging.version>\${dep.airlift.version}</dep.packaging.version>
+        <dep.errorprone.version>2.14.0</dep.errorprone.version>
+        <dep.jackson.version>2.13.3</dep.jackson.version>
+        <dep.testng.version>7.6.0</dep.testng.version>
+
+        <air.javadoc.lint>-missing</air.javadoc.lint>
+    </properties>
+
+    <dependencyManagement>
```

Comment:
Why do we need these? Nothing in an example plugin should use JNA

Reply:
It's not specified in Airbase, only in Trino. Should I add it to Airbase?
```
[INFO] [WARNING] Rule 2: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
[INFO] Failed while enforcing RequireUpperBoundDeps. The error(s) are [
[INFO] Require upper bound dependencies error for net.java.dev.jna:jna:5.8.0 [provided] paths to dependency are:
[INFO] +-com.example:trino-example-plugin:0.1-SNAPSHOT
[INFO]   +-io.airlift:configuration:217
[INFO]     +-net.bytebuddy:byte-buddy:1.12.6
[INFO]       +-net.java.dev.jna:jna:5.8.0 [provided]
[INFO] and
[INFO] +-com.example:trino-example-plugin:0.1-SNAPSHOT
[INFO]   +-io.trino:trino-main:391 [test]
[INFO]     +-com.github.oshi:oshi-core:5.8.5 [test]
[INFO]       +-net.java.dev.jna:jna:5.10.0 [test]
[INFO] and
[INFO] +-com.example:trino-example-plugin:0.1-SNAPSHOT
[INFO]   +-io.trino:trino-main:391 [test]
[INFO]     +-net.java.dev.jna:jna-platform:5.10.0 [provided]
[INFO]       +-net.java.dev.jna:jna:5.10.0 [provided]
[INFO] ]
```

Reply:
I opened https://github.com/airlift/airlift/pull/1004

Reply:
Done!